cloud-formation/account-frontend-cf.yaml

```diff
@@ -227,11 +230,11 @@ Resources:
         - IpProtocol: tcp
           FromPort: 22
           ToPort: 22
-          CidrIp: 77.91.248.0/21
+          CidrIp: !Ref OfficeCIDR
         - IpProtocol: tcp
```

Comment:
Note, we should get rid of this direct box access once we're happy with things and before we go live-live.